jsonLoader.go

```diff
@@ -279,6 +287,7 @@ func (l *jsonGoLoader) LoaderFactory() JSONLoaderFactory {
 	return &DefaultJSONLoaderFactory{}
 }
 
+// NewStringLoader creates a new JSONLoader for the given source
```

Comment:
This should be something along the lines of:
"`NewGoLoader` creates a new JSONLoader from a given Go struct"

Reply:
👍 good suggestion; updating 🤗